doc/api/cli.md

```diff
@@ -6,6 +6,9 @@
 Node.js comes with a variety of CLI options. These options expose built-in
 debugging, multiple ways to execute scripts, and other helpful runtime options.
 
+Worker threads inherit non-process-specific options by default. Refer to
```

Comment:
Is this a good place for this information? This is in a very prominent place in this document but it only applies to worker threads. This page is the documentation I would look at to figure out what CLI options I want to use. For what worker threads inherit, I would look in the `worker_threads` docs or maybe at `process.argv` and friends.

It took me a while to figure out what `non-process-specific options` means here, but I'm also not coming up with better terminology.

Reply:
@Trott ,thanks. Added in worker_threads.